index.html

```diff
@@ -0,0 +1,634 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contributing to Open Source Part I: The Easy Way - YouTube</title>
+    <link rel="icon" href="image/youtube.png" type="image/png">
+    <meta name="author" content="C. Miguel V. Vega">
+    <meta name="description" content="Embedding Images and Video.">
+    <meta name="keywords" content="HTML, CSS">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="flex-container">
+            <div class="nav-icon">
+                <button class="material-icons">menu</button>
+            </div>
+
+            <div class="navbar-logo">
+                <img src="image/yt_logo_rgb_light.png"
```

Comment:
`SUGGESTION`

Consider adding the images you referenced. You seem to be adding images that are not available on the repo.